apps/src/templates/progress/ProgressBubbleSet.jsx

```diff
@@ -84,7 +84,7 @@ const ProgressBubbleSet = React.createClass({
                 <ProgressPill
                   url={level.url}
                   status={level.status}
-                  text={i18n.unpluggedActivity()}
+                  text={level.isGoBeyond ? i18n.goBeyond() : i18n.unpluggedActivity()}
```

Comment:
are go beyond levels a subtype of unplugged? (we only hit this branch if level.isUnplugged)